File: python/5 kyu/112.py
# https://www.codewars.com/kata/5266fba01283974e720000fa/train/python

# Passed

def variance(numbers): 
    mean = sum(numbers)/len(numbers)
    square_of_differences = [(num - mean) ** 2 for num in numbers]
    variance = sum(square_of_differences)/len(numbers) 
    return variance

output = variance([8, 9, 10, 11, 12])
print(output)

File: nodejs/7 kyu/112.js
// https://www.codewars.com/kata/566044325f8fddc1c000002c/train/javascript

// Passed

function evenChars(string) {
  if (string.length < 2 || string.length > 100) return "invalid string";

  let evenChars = [];
  for (let idx = 1; idx < string.length; idx += 2) {
    let char = string[idx];
    evenChars.push(char);
  }
  return evenChars;
}

const output = evenChars("abcdefghijklm");
console.log(output);